InTune/InTune/ViewControllers/ProfileViewController.swift

```diff
@@ -48,6 +50,8 @@ class ProfileViewController: UIViewController {
         }
         
         emailLabel.text = "\(email)"
+        likeArtistButton.isHidden = true
```

Comment:
can use mvvm and create a viewModel struct and move this hide logic from here to that struct.

```
struct ProfileViewMode {

var shouldShowLikeArtistButton: Bool {
// until you have the business logic to determine you can return true as you are doing in the view controller

return true
}
```